src/main/java/com/growup/pms/user/controller/dto/request/UserUpdateRequest.java

```diff
@@ -3,36 +3,39 @@
 import static com.growup.pms.common.constant.RegexConstants.NICKNAME_PATTERN;
 
 import com.growup.pms.user.service.dto.UserUpdateCommand;
-import jakarta.validation.constraints.NotNull;
 import jakarta.validation.constraints.Pattern;
-import jakarta.validation.constraints.Size;
-import java.util.List;
+import lombok.AccessLevel;
 import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
 import org.hibernate.validator.constraints.Length;
-import org.hibernate.validator.constraints.URL;
+import org.openapitools.jackson.nullable.JsonNullable;
 
-@Builder
-public record UserUpdateRequest(
-        @NotNull
-        @Pattern(regexp = NICKNAME_PATTERN)
-        String nickname,
+@Getter
+@NoArgsConstructor(access = AccessLevel.PRIVATE)
+public class UserUpdateRequest {
+    @Pattern(regexp = NICKNAME_PATTERN)
+    JsonNullable<String> nickname = JsonNullable.undefined();
 
-        @NotNull
-        @Length(max = 200)
-        String bio,
+    @Length(max = 300)
```

Comment:
이 부분은 원래는 300자였는데 프론트 분이랑 논의 후에 200자로 통일되어서 아래와 같이 수정되면 좋을 것 같아요! 그리고 `@Length`가 Hibernate 애노테이션이고, `@Size`는 자바 표준 애노테이션으로 알고 있는데 후자가 이식성이 좋다고 해서 검증 애노테이션을 후자로 통일해보면 좋을 것 같습니다 :thinking: 

```suggestion
    @Size(max = 200)
```

![image](https://github.com/user-attachments/assets/61678883-517a-4257-8143-1194b5096fca)

Reply:
https://www.baeldung.com/jpa-size-length-column-differences